backend-project/small_eod/migration_v1/README.md

```diff
@@ -0,0 +1,44 @@
+
+###Bootstrap on dev:
+
+To run migrator locally:
+```shell script
+docker-compose up -d migration_db
+docker-compose exec migration_db mysql -ppass --execute 'CREATE DATABASE small_eod CHARACTER SET utf8;'
+docker-compose exec -T migration_db mysql small_eod -ppass < ./small_eod_demo-26-09-2020.sql
+docker-compose up -d --build
+docker-compose run --rm backend python manage.py migrate_v1
+```
+
+### Added unique constraints:
+
+I added some simple constraints that seemed missing from the data model:
+ - Channel.name
+ - Feature.name
+ - FeatureOption.('name', 'feature')
+ - DocumentType.name
+ - Tag.name
+
+### Migration assumptions:
+
+I had to make some assumptions about "what(v1)-goes-where(v2)" that may or may not be accurate:
+ - Dictionary.type is now Feature.name
+ - Dictionary.name is now FeatureOption.name
+ - LetterName model is now DocumentType
+ - Users responsible for case will want be notified
+ - Letter.identifier is now Letter.reference_number
+ 
+### Issues:
+ - v1 has no data for created_by and modified_by fields in general
+ - Case.comment field too short for source data
+ - Should Feature.name be translated?
+ - Should we subtract 1 while migrating Letter ordering?
```

Comment:
I don't understand, but see #557 . I am afraid that regard sorting we may have new constraint and other changes when the element starts to be used in the front-end.

Reply:
I'm not that familiar with ordering from the domain PoV, but this question spawns directly from the change in default value for ordering field between v1 and v2.
 - v1 default=1 [here](https://github.com/watchdogpolska/small_eod/blob/master/small_eod/cases/models.py#L223)
 - v2 default=0 [here](https://github.com/watchdogpolska/small_eod/blob/dev/backend-project/small_eod/letters/models.py#L44)

Reply:
I don't think it matters. It's just that no one has noticed this inconsistency before. At the moment, I propose to standardize the model in order not to complicate the migration.